src/modules/audio.c

```diff
@@ -70,6 +99,8 @@ AUDIO_allocate(WrenVM* vm) {
     SDL_FreeWAV((uint8_t*)tempBuffer);
   } else if (data->audioType == AUDIO_TYPE_OGG) {
     free(tempBuffer);
+  } else if (data->audioType == AUDIO_TYPE_MP3) {
+    free(tempBuffer);
   }
```

Comment:
Shouldn't there a case for FLAC, too?

Reply:
You're right. I was just adding it!

Reply:
Ah yes, just a couple of minutes ago :-)
Sorry about that.

Reply:
Not at all, I appreciate the oversight. I wrote/merged a lot of code today so mistakes are likely.